app/static/src/app/store/fitData/actions.ts

```diff
@@ -35,15 +37,25 @@ const updateLinkedVariables = (context: ActionContext<FitDataState, FitState>) =
     commit(FitDataMutation.SetLinkedVariables, newLinks);
 };
 
+const updateTimeVariable = (context: ActionContext<FitDataState, FitState>) => {
```

Comment:
Could rename this as it's in response to the time variable being updated- it doesn't do that update itself. Though that makes it less of a  consistent pair with `updateLinkedVariables` (which actually does update the linked variables)
```suggestion
const timeVariableUpdated = (context: ActionContext<FitDataState, FitState>) => {
```

Reply:
ok, I've gone with `respondUpdatedTimeVariable` so that it's verby